modules/images/dominant-color/load.php

```diff
@@ -126,8 +126,12 @@ function dominant_color_img_tag_add_dominant_color( $filtered_image, $context, $
 		$extra_class  = $image_meta['has_transparency'] ? 'has-transparency' : 'not-transparent';
 	}
 
-	if ( ! empty( $data ) || ! empty( $style ) ) {
-		$filtered_image = str_replace( '<img ', '<img ' . $data . $style, $filtered_image );
+	if ( ! empty( $data ) || ! empty( $style ) || ! str_contains( $data, $filtered_image ) ) {
+		if ( strpos( $filtered_image, 'style=' ) ) {
```

Comment:
Suggest using `str_contains()` instead of `strpos()`.

Reply:
yes swaped